config/config.go

```diff
@@ -474,9 +474,10 @@ func (c *Config) Parse() error {
 		flag.Parse()
 	}
 
-	if c.ApiUsageMonitoringDefaultClientTrackingPattern != "" {
-		log.Warn(`"api-usage-monitoring-default-client-tracking-pattern" parameter is deprecated`)
-	}
+	c.deprecated("enable-prometheus-metrics", false)
+	c.deprecated("api-usage-monitoring-default-client-tracking-pattern", "")
+	c.deprecated("enable-kubernetes-east-west", false)
+	c.deprecated("kubernetes-east-west-domain", "")
```

Comment:
This is a simple but requires repetition of option name and a default value. Here is a bit more involved variant https://github.com/zalando/skipper/tree/deprecated-flag but I favor simplicity

Reply:
I've changed the implementation to check if flag/config is set regardless of the value